jvm/sargon-android/src/main/java/com/radixdlt/sargon/extensions/AccountAddress.kt

```diff
@@ -7,7 +7,7 @@ import com.radixdlt.sargon.PublicKey
 import com.radixdlt.sargon.accountAddressBech32Address
 import com.radixdlt.sargon.accountAddressIsLegacy
 import com.radixdlt.sargon.accountAddressNetworkId
-import com.radixdlt.sargon.accountAddressToShort
+// import com.radixdlt.sargon.accountAddressFormatted
```

Comment:
I did not get this working, could not import it. What do I need to do to get it working? I ran `./jvm/gradlew -p jvm/sargon-android testDebugUnitTest` which I thought would build allowing me to import...

anyway, I will leave it like this, since you will wanna add the address formatting on all address types in Kotlin after this PR has been merge anway...

Reply:
No problem I will add it.